docs/src/appendix/sncast-library/fee_settings_trait.md

```diff
@@ -0,0 +1,49 @@
+# `FeeSettingsTrait`
```

Comment:
I think maybe just showing the signatures and explaining their intended usages would be better. Let's treat the returned struct as an implementation detail.